Clicker/ViewControllers/PollBuilderViewController.swift

```diff
@@ -19,41 +19,40 @@ protocol PollBuilderDelegate {
 
 class PollBuilderViewController: UIViewController, QuestionDelegate, PollBuilderDelegate, FillsDraftDelegate, PollTypeDropDownDelegate{
     
+    // MARK: layout constants
     let questionTypeButtonWidth: CGFloat = 150
     let draftsButtonWidth: CGFloat = 100
     let popupViewHeight: CGFloat = 95
-    var pickQTypeVC: PickQTypeViewController!
-    
-    var dropDown: PollTypeDropDownView!
-    var dropDownArrow: UIImageView!
-    
     let edgePadding: CGFloat = 18
     let topBarHeight: CGFloat = 24
     let dropdownArrowHeight: CGFloat = 5.5
     let buttonsViewHeight: CGFloat = 67.5
     let buttonHeight: CGFloat = 47.5
-    
+
+    // MARK: subviews and VC's
+    var pickQTypeVC: PickQTypeViewController!
```

Comment:
i checked the file and this var is unused. can you remove this if we're not using it?